src/enums/roleTypes.ts

```diff
@@ -4,5 +4,6 @@ export enum RoleTypes {
   DIRECTOR = 'Director',
   SUBSCRIBER = 'Subscriber',
   LIQUIDATOR = 'Liquidator',
-  CUSTODIAN = 'Custodian'
+  CUSTODIAN = 'Custodian',
+  STAFF = 'STAFF'
```

Comment:
Why a different case for staff?

Reply:
Mostly to reference an enum value in the logic vs a string. 
This enum made sense as we refer to `staff` as a role type, although not directly in the context of a specific filing as some of the other roles are.
We could declare it as a local const as well, either or.

Reply:
Should the other items in this list be all caps then? Or is this enum used for 2 different things?

Reply:
I think it's a matter of what we are comparing it too/from. 
`STAFF` is all caps as thats how the value is represented in the `userInfo` response.
In the other use cases RoleTypes is being used as both value and label. If i had to guess the value might not matter if it's all caps but the Label might not want to be all caps. 
So yea, two different cases. 
Perhaps it makes sense to declare a local const of our `STAFF` role for use in the logic vs put it in this enum? 
The use cases do differ so the value format wouldn't really be cohesive with the others.

Reply:
I recommend a local const for STAFF.

Reply:
done and done, ty!